### PR TITLE
PR for week of May 13-17

### PR DESCRIPTION
**Pull request by connorpheraty**

### CODE REVIEW

**Review by veritaem — COMMENTED** [review]
your model is solid but in order to make more impressive results you will want to engineer more creative features.  For instance, could you create a wind chill, heat index, or boolean flags that have some threshold, like warm season/cold season, or season in general, etc?

**Review by veritaem — COMMENTED** [review]
not too bad, although I might add that as we only have 70 months of data, it could be that the signifigant dropoff at 60+ months is due to them just having some scandal, or people naturally moving their lives around(i dont remember if this talked about location but imagine the churn in a college town vs a farm...place?).  Either way great job, any questions for standup?

**Review by veritaem — COMMENTED** [review]
Your pdp plots and things look very nice but If I may quibble with a couple things, specifically your 'model.fit' and your train test split?

 *  because the data in the df is ordered by date, and you just split it without shuffling, you lose a great deal of time series info (are more ppl buying bikes? Is pollution going up?). train_test_split has a default shuffle, it may help?
 *  because you are estimating the best scoring model on your training data, the best score is just the one that fits all of your already labeled points (which is why its max depth/max trees), which will lead to some overfitting and a lesser prediction score. Have you tried making the model with a train score and predicting the val score to get your best model out that way?
